games-list/src/components/OneGame.tsx

```diff
@@ -57,20 +91,31 @@ export default function OneGame({ element, gamesList, setList }: OneGameProps) {
     >
       <div className={styles.innerBox}>
         <span
-          onClick={() => completeCheck(element)}
-          className={getName(element.complete)} //style changes depending on complete status (strikethrough)
+          onClick={() => completeCheck(gameObject)}
+          className={getName(gameObject.complete)} //style changes depending on complete status (strikethrough)
         >
-          {element.game /* name of game to conquer */}
+          {gameObject.game /* name of game to conquer */}
         </span>
         <button
-          onClick={() => revealDetails(element)}
+          onClick={() => revealDetails(gameObject)}
           className={styles.reveal}
         >
           v
         </button>
-        <button onClick={() => handleClick(element)} className={styles.delete}>
+        <button
+          onClick={() => closeObject(gameObject)}
+          className={styles.delete}
+        >
           x
         </button>
+        <Link //takes us to new page which describes the game
+          href={{
+            pathname: "/GameDescription",
+            query: { gameName: JSON.stringify(gameObject) },
+          }}
+        >
+          <button className={styles.newPage}>↗</button>
+        </Link>
```

Comment:
Instead of routing to `GameDescription` with a query parameter, lookin into Nextjs dynamic routes. :) https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes

Also, convention states that page names are kebab-case ("game-description" in this... ah, case)